operator-common/src/test/java/io/strimzi/operator/common/operator/resource/KafkaConnectCrdOperatorIT.java

```diff
@@ -305,7 +304,7 @@ public void testUpdateStatusWhileResourceUpdated(TestContext context)    {
         log.info("Updating resource");
         KafkaConnect updated = new KafkaConnectBuilder(kafkaConnectOperator.get(namespace, RESOURCE_NAME))
                 .editSpec()
-                .withLogging(new InlineLogging())
+                .withNewLivenessProbe().withInitialDelaySeconds(14).endLivenessProbe()
```

Comment:
is this related to this PR or you just noticed that? and why removing logging and adding probes?

Reply:
Well in testing this PR I discovered that the tests failed on minikube with kubernetes 1.14, but passed on `oc cluster up`. The tests didn't fail before on Travis because Travis was using 1.9, but these tests exit early when kubernetes <1.11 is detected.

I _think_ the reason the the logging change doesn't work is that, for whatever reason, kubernetes doesn't see the patch as causing an update. In certainly doesn't result in the resourceVersion getting incremented, which is what the test depends on (ultimately we're trying to get an resource version conflict).